sql/connect/common/src/main/protobuf/spark/connect/base.proto

```diff
@@ -261,6 +261,10 @@ message AnalyzePlanResponse {
 
   message SparkVersion {
     string version = 1;
+    // The java version the Spark Connect Service is built with.
```

Comment:
Is this correct, @garlandz-db ?

Reply:
Agree this is the JRE version not the built version. Same is true for the Scala version likely.